### ISSUE TITLE
v1 example showing data from map

### ISSUE THREAD

**Issue by tomwayson** [Esri/cedar#402]
The purpose of this example would be to demonstrate how to use inlined data, similar to this [v0 example](http://esri.github.io/cedar/examples/inlined-data.html). However, in this case, I'd like the inlined data to come from the features in a map layer, since that's the probably most common use case for inline data.

I'd like to use the same service that's in the [codepen](https://codepen.io/tomwayson/pen/paxgeO?editors=0010#0) from #339:

https://maps7.arcgisonline.com/arcgis/rest/services/Veterans_Proportion_From_Population/MapServer/0

That service even has a renderer defined, so we can probably use it's default renderer when adding it to the map. It looks like that renderer uses a different field (`Grand_Tota`) than the above codepen (`TotPop`), so if we do use that renderer, we should also use the the `Grand_Tota` field for the chart.

When the layer finishes loading, get the features, and pass them to cedar using something like:

```
var definition = {
  datasets: [{
    data: {
      features: features
    }
  }]
};
```

**Comment by jPurush** [comment]
@tomwayson should this chart respond to extent changes in the map as well?

**Comment by tomwayson** [comment]
Another thing

I assume the features will not be sorted when you get them from the layer and the chart will look like: 

![image](https://user-images.githubusercontent.com/662944/36936179-4e789c24-1eb6-11e8-9e9f-c722a95ccac7.png)

I don't think there's a way to set a sort expression on the layer definition, but we can sort the array of features before passing to cedar w/ [`Array.prototype.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). However, that sorts in place - I don't know if that's going to be a problem (like did the feature layer pass a _copy_ of the feature's graphics, or a direct reference, and if it's the latter, will it mess up the map if we sort them). We should try sorting directly, but If that is a problem, we can always [make a copy before sorting](https://stackoverflow.com/a/9592755/656010).

**Comment by jPurush** [comment]
So I was trying to decide between a query task and a layer 'view' query - this helps! And [mode](https://developers.arcgis.com/javascript/latest/guide/functionality-matrix/index.html#featurelayer) does not exist in 4.x.

**Comment by tomwayson** [comment]
Yeah, I think for this example we're trying to show the "I already have the features, I should not need to make another request" - tho, I noticed that the layer view's `queryFeatures()` returns a promise, so who knows what it's doing under the hood. Maybe it uses some smarts to deal w/ the "mode" issues.

The only reason to do a query task rather than pass `url` and `query` to cedar would be to take advantage of the integrated auth w/ `IdentityManager`. That is a valid reason, but if we built such an example it would have to show that (i.e. use a secure service).
